internal/librarian/release_init.go

```diff
@@ -262,16 +274,13 @@ func (r *initRunner) runInitCommand(ctx context.Context, outputDir string) error
 //
 // 4. Set the library's release trigger to true.
 func (r *initRunner) updateLibrary(library *config.LibraryState) error {
-	// Update the previous version, we need this value when creating release note.
-	library.PreviousVersion = library.Version
 	commits, err := GetConventionalCommitsSinceLastRelease(r.repo, library)
 	if err != nil {
 		return fmt.Errorf("failed to fetch conventional commits for library, %s: %w", library.ID, err)
 	}
 
-	library.Changes = commits
-	if len(library.Changes) == 0 {
-		slog.Info("Skip releasing library since no eligible change is found", "library", library.ID)
+	if len(commits) == 0 {
```

Comment:
We shouldn't skip this if we've been asked explicitly to release a single library. See https://github.com/googleapis/librarian/issues/2032

You could potentially pass a Boolean flag into this function as a "force even if there are no commits".

Reply:
Thanks with Jon offline and we're fine with making the changes in #2030 in a separate PR.